src/Nethermind/Nethermind.Monitoring/Config/MetricsConfig.cs

```diff
@@ -8,6 +8,8 @@ public class MetricsConfig : IMetricsConfig
         public string ExposeHost { get; set; } = "+";
         public int? ExposePort { get; set; } = null;
         public bool Enabled { get; set; } = false;
+
+        public bool PushAfterBlock { get; set; } = false;
```

Comment:
If idea here would work I'd probably prefer it to be enabled by default - this way we have much better view on metrics than now.